antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -1111,7 +1113,10 @@ def _clear_all_snapshots(self) -> None:
                         self._variant_study_service.clear_snapshot(variant)
 
     def run_task(self, notifier: TaskUpdateNotifier) -> TaskResult:
-        msg = f"Start cleaning all snapshots updated or accessed {self._retention_hours} hours ago."
+        msg = (
+            f"Start cleaning all snapshots updated or accessed"
+            f" {self._retention_hours.total_seconds() // 3600} hours ago."
+        )
         notifier(msg)
         self._clear_all_snapshots()
         msg = f"All selected snapshots were successfully cleared."
```

Comment:
Remove this useless f-string. Also I prefer the solution that Sylvain proposed
Also you should apply this change inside the variant blueprint you've introduced the same log there